File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int m=s.size(); int n=t.size();
        if(n!=m)
        return false;
        unordered_map< char,int> s1;
        unordered_map< char,int> s2;
        int count =0;
        for(int i =0; i <m ;i++)
        {
            s1[s[i]]++;
        } 
        for(int i =0; i <m ;i++)
        {
            s2[t[i]]++;
        } 
        for(int i =0;i<m ;i++)
        {
            if(  s1[s[i]] !=  s2[s[i]]  )
            return false;
        }
       
        return true; 
    }  
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res = []
        count = {}
        for n in nums:
            if n in count:
                count[n]+=1
            else:
                count[n]=1
        new_count = sorted(count.items(), key=lambda pair: pair[1], reverse = True)
        for i in range(k):
            res.append(new_count[i][0])
        return res